### PR TITLE
feat(websocket): add ws instrumentation

### PR DESCRIPTION
**Pull request by idonava**
1. Added ws instrumentation to incoming messages.
2. Upgrade epsagon version.

### CODE REVIEW

File: src/wrappers/ws.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+ * @param {*} requestFunctionThis request arguments.
+ */
+function websocketEmitterMiddleware(message, originalHandler, requestFunctionThis) {
+    let originalHandlerSyncErr;
+
+    try {
+        // Initialize tracer and evnets.
+        tracer.restart();
+        const { slsEvent: websocketEvent, startTime: websocketStartTime } =
+        eventInterface.initializeEvent(
```
Comment (galbash) [inline src/wrappers/ws.js:33]: indent?

**Comment by ranrib** [comment]
Please update readme here and in the node one

File: src/wrappers/ws.js (hunk 1 above)
Comment (idonava) [reply src/wrappers/ws.js:33]: yes

**Pull request merged**